tests/std/tests/P0784R7_library_support_for_more_constexpr_containers/test.cpp

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <assert.h>
+#include <memory>
+#include <stddef.h>
+#include <string.h>
+#include <string>
+#include <type_traits>
+#include <utility>
+
+using namespace std;
+
+template <class Void, class Ty, class... Types>
+inline constexpr bool can_construct_at_impl = false;
+
+template <class Ty, class... Types>
+inline constexpr bool
+    can_construct_at_impl<void_t<decltype(construct_at(declval<Ty*>(), declval<Types>()...))>, Ty, Types...> = true;
+
+template<class Ty, class... Types>

```

Comment:
This is failing clang-format. I recommend configuring your editor to format on save.

Reply:
Unfortunately I can't use format-on-save because I spend too much time editing things that aren't the STL that are damaged when I do that :(

Reply:
(I'm quite happy to have our CI yell at me :) )

Reply:
Please consider using `code .` as https://github.com/microsoft/STL/blob/d4ee5c3fb4abfe8492f2108384d6f4e0fd0afeae/.vscode/settings.json#L4 will set format-on-save for just this workspace.